addons/website/static/src/js/menu/seo.js

```diff
@@ -192,12 +192,13 @@ var KeywordList = Widget.extend({
             keyword.on('selected', self, function (word, language) {
                 self.trigger('selected', word, language);
             });
-            keyword.appendTo(self.$el);
+            keyword.appendTo(self.$el).then(() => {
+                self.trigger('content-updated');
+            });
```

Comment:
Diff would be smaller and code probably better if you make the function `async` and add `await` before this `appendTo` this would ensure that all remaining code is done after this `appendTo`.